types/api/xverse/ordinals.ts

```diff
@@ -38,25 +38,84 @@ export interface InscriptionInCollectionsList {
   data: Array<Inscription>;
 }
 
-export type SatRarity = 'common' | 'uncommon' | 'rare' | 'epic' | 'legendary' | 'mythic';
+export const RoadArmorRareSats = ['MYTHIC', 'LEGENDARY', 'EPIC', 'RARE', 'UNCOMMON', 'COMMON'] as const;
+export type RoadArmorRareSatsType = (typeof RoadArmorRareSats)[number];
 
-export type BundleRareSat = {
-  number: string;
-  rarity_ranking: SatRarity;
-  offset: number;
-};
+export const Sattributes = [
+  'BLACK_LEGENDARY',
+  'BLACK_EPIC',
+  'BLACK_RARE',
+  'BLACK_UNCOMMON',
+  'FIBONACCI',
+  '1D_PALINDROME',
+  '2D_PALINDROME',
+  '3D_PALINDROME',
+  'SEQUENCE_PALINDROME',
+  'PERFECT_PALINCEPTION',
+  'PALIBLOCK_PALINDROME',
+  'PALINDROME',
+  'NAME_PALINDROME',
+  'ALPHA',
+  'OMEGA',
+  'FIRST_TRANSACTION',
+  'BLOCK9',
+  'BLOCK78',
+  'NAKAMOTO',
+  'VINTAGE',
+  'PIZZA',
+  'JPEG',
+  'HITMAN',
+  'SILK_ROAD',
+] as const;
+export type SattributesType = (typeof Sattributes)[number];
+
+// ({} & string) is a workaround to support our types and also allow any string for unsupported types
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type RareSatsType = RoadArmorRareSatsType | SattributesType | ({} & string);
```

Comment:
Hmmm.... isn't this just the same as `export type RareSatsType = string;` ?

Reply:
nop 
![Screenshot 2023-11-29 at 12 51 24](https://github.com/secretkeylabs/xverse-core/assets/18606335/095f1590-49d7-48ec-b67c-ad6c1ec94b4c)